### DISCUSSION TITLE
REFPROP Fluids List

### DISCUSSION THREAD

**Discussion by artsinli** [CoolProp/CoolProp#2216] [Q&A]
I know that you could use the REFPROP backend to calculate properties, but is there a way to access the fluids list from REFPROP and use that instead?

Something like this:
```
cp.set_config_string(cp.ALTERNATIVE_REFPROP_PATH, "C:\\Program Files (x86)\\REFPROP")
REFPROP = cp.AbstractState('REFPROP', 'R454B')
```
Gives this error: 
`ValueError: Could not load these fluids: R454B`

**Comment by ibell** [comment]
Try with ``R454B.MIX`` as the fluid name

**Reply by JawadJaber** [reply]
Why I try to get the properties of R454B.Mix, there is an exception raised. So, we are looking for another source for this refrigerant.
Pls let me know for additional information?

**Comment by ibell** [comment]
I assume you are not on windows?  There is a big with refprop preventing
the parsing of predefined mixtures on non windows platforms.  So you'll
need to provide the components and composition from the predefined mixture
file.

**Reply by JawadJaber** [reply]
No it is windows, try using SharpFluids repo and use this refrigerant.

https://github.com/MadsKirkFoged/SharpFluids

And follow the issue at the following link

https://github.com/MadsKirkFoged/SharpFluids/issues/49#issue-1718423220

**Comment by ibell** [comment]
Please show the code you tried and what error you obtained.
